### PR TITLE
Add Palemonas font

### PR DESCRIPTION
**Pull request by tghs**
Details in English: http://www.mif.vu.lt/~vladas/Palemonas.htm

### CODE REVIEW

File: Casks/font-palemonas.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'font-palemonas' do
+  version '3.0'
+  sha256 'ad576fc82d1f67824cc10cde6c1778016ecf16702bcc2353f0d3884f2283a978'
+
+  url 'http://www.vlkk.lt/media/public/file/Palemonas/Palemonas-3_0.zip'
+  homepage 'http://www.vlkk.lt/palemonas'
+  license :gratis
+  container :type => :zip
```
Comment (vitorgalvao) [inline Casks/font-palemonas.rb]: Are you sure this is needed? Shouldn’t be.

File: Casks/font-palemonas.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'font-palemonas' do
+  version '3.0'
+  sha256 'ad576fc82d1f67824cc10cde6c1778016ecf16702bcc2353f0d3884f2283a978'
+
+  url 'http://www.vlkk.lt/media/public/file/Palemonas/Palemonas-3_0.zip'
```
Comment (vitorgalvao) [inline Casks/font-palemonas.rb]: `url "http://www.vlkk.lt/media/public/file/Palemonas/Palemonas-#{version.sub('.','_')}.zip"`.

Comment (tghs) [inline Casks/font-palemonas.rb]: Thanks, I'll resubmit with that change squashed in.

File: Casks/font-palemonas.rb (hunk 1 above)
Comment (tghs) [inline Casks/font-palemonas.rb]: Yes, I initially tried without that. But then you get this error:

```
Error: Uh oh, could not identify primary container for '/Library/Caches/Homebrew/font-palemonas-3.0.zip'
```

I've not dug any further into why the primary container detection is getting tripped up by this file.

Comment (vitorgalvao) [inline Casks/font-palemonas.rb]: Sounds good. Leave it, then.

**Pull request closed without merging**